include/deal.II/base/geometry_info.h

```diff
@@ -23,6 +23,179 @@
 
 DEAL_II_NAMESPACE_OPEN
 
+#ifndef DOXYGEN
+namespace internal
+{
+  // A struct that holds the values for all the arrays we want to initialize in GeometryInfo
+  template <int dim>
+  struct initializers;
```

Comment:
please move these classes into a further namespace `internal::GeometryInfo` to make sure we're not stepping onto similar classes elsewhere. Also maybe capitalize the class name?

Reply:
I've chosen `internal::GeometryInfoHelper`. Using `internal::GeometryInfo` is not a good idea since we are accessing GeometryInfo also inside `internal` namespaces and then `namespace GeometryInfo` rather than `class GeometryInfo` is picked up.

Reply:
Fair point.